### PR TITLE
--display-ignore-fields in display_model.

### PR DESCRIPTION
**Pull request by stephenroller**

### CODE REVIEW

File: examples/display_model.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ def main():
     # Get command line arguments
     parser = ParlaiParser(True, True)
     parser.add_argument('-n', '--num-examples', default=10)
+    parser.add_argument('--display-ignore-fields', type=str, default='agent_reply')
```
Comment (jaseweston) [inline examples/display_model.py]: group arguments?

**Comment by alexholdenmiller** [comment]
wait won't that default remove the model's predictions from the display?

**Comment by stephenroller** [comment]
@alexholdenmiller no

but i'll remove the default

https://github.com/facebookresearch/ParlAI/blob/728b763d70a1eff5aa190f492d30bff83eee68c0/parlai/core/utils.py#L791-L800

File: examples/display_model.py (hunk 1 above)
Comment (stephenroller) [reply examples/display_model.py]: ?

**Comment by alexholdenmiller** [comment]
https://github.com/facebookresearch/ParlAI/blob/728b763d70a1eff5aa190f492d30bff83eee68c0/parlai/core/utils.py#L778-L781

doesn't that skip the agent?

**Comment by stephenroller** [comment]
I've put the argument in, and removed the `agent_reply` bug; then linted. It's ready for a second review.

**Comment by stephenroller** [comment]
Rebased, waiting on tests to merge.

**Pull request merged**